tldraw/apps/tldraw-logseq/package.json

```diff
@@ -33,7 +33,9 @@
     "tsup": "^6.1.2",
     "typescript": "^4.7.3",
     "zx": "^6.2.4",
-    "@radix-ui/react-switch": "^0.1.6-rc.40"
+    "@radix-ui/react-switch": "^0.1.6-rc.40",
+    "@radix-ui/react-dropdown-menu": "^0.1.6",
+    "@stitches/react": "^1.2.8"
```

Comment:
I haven't made a decision which css library to use. How do you pick stitches over other choices?

Reply:
It was actually just one I was comfortable with, I can switch it over to static CSS if that's better.

Reply:
Please use plain css for now. This is something we haven't decided yet.

Reply:
I have Reverted to plain css